tests/lean/run/recommendedSpelling.lean

```diff
@@ -0,0 +1,64 @@
+import Lean
+
+/-!
+Test the `recommended_spelling` command.
+
+TODO: once we use this command in Init, we can test that recommended spellings from imported
+modules are reported correctly.
+-/
+
+/--
+Conjuction
+
+Second line
+-/
+inductive MyAnd (P Q : Prop) : Prop where
+  | intro : P → Q → MyAnd P Q
+
+@[inherit_doc] infixr:35 " ☃ " => MyAnd
+@[inherit_doc] infixr:35 " ☋ " => MyAnd
+
+recommended_spelling "☃" "snowmand" MyAnd «term_☃_»
```

Comment:
This spelling convention is a source of delight and wonder, and I hope it remains in the test suite forever